test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -930,102 +928,109 @@ public void VerifyExpectedAndActualBooleanOptionsDictionary(Dictionary<string, D
-            var model = op.GenerateTSqlModel();
+        GenerateTSqlModelOperation op = new GenerateTSqlModelOperation(generateTSqlScriptParams);
+        var model = op.GenerateTSqlModel();
 
-            // clean up
-            Directory.Delete(TSqlModelTestFolder);
-
-            Assert.That(model.GetObjects(DacQueryScopes.UserDefined, ModelSchema.Table).ToList().Count, Is.EqualTo(0), "Model is not empty");
-            Assert.That(model.Version.ToString(), Is.EqualTo(generateTSqlScriptParams.ModelTargetVersion), $"Model version is not equal to {generateTSqlScriptParams.ModelTargetVersion}");
-        }
+        Assert.That(model.GetObjects(DacQueryScopes.UserDefined, ModelSchema.Table).ToList().Count, Is.EqualTo(0), "Model is not empty");
+        Assert.That(model.Version.ToString(), Is.EqualTo(generateTSqlScriptParams.ModelTargetVersion), $"Model version is not equal to {generateTSqlScriptParams.ModelTargetVersion}");
+    }
 
-        /// <summary>
-        /// Verify the generate TSql Model handle
-        /// </summary>
-        [Test]
-        public async Task VerifyGenerateTSqlModelHandle()
+    /// <summary>
+    /// Verify the generate TSql Model handle
+    /// </summary>
+    [Test]
+    public async Task VerifyGenerateTSqlModelHandle()
+    {
+        var generateTSqlScriptParams = new GenerateTSqlModelParams
         {
-            DacFxService service = new DacFxService();
-            Directory.CreateDirectory(TSqlModelTestFolder);
+            ProjectUri = Path.Join(TSqlModelTestFolder, "test.sqlproj"),
+            ModelTargetVersion = "Sql160",
+            FilePaths = new string[] { }
+        };
 
-            var generateTSqlScriptParams = new GenerateTSqlModelParams
-            {
-                ProjectUri = Path.Join(TSqlModelTestFolder, "test.sqlproj"),
-                ModelTargetVersion = "Sql160",
-                FilePaths = new string[] { }
-            };
+        var requestContext = new Mock<RequestContext<bool>>();
+        requestContext.Setup((RequestContext<bool> x) => x.SendResult(It.Is<bool>((result) => result == true))).Returns(Task.FromResult(new object()));
 
-            var requestContext = new Mock<RequestContext<ResultStatus>>();
-            requestContext.Setup((RequestContext<ResultStatus> x) => x.SendResult(It.Is<ResultStatus>((result) => result.Success == true))).Returns(Task.FromResult(new object()));
-            await service.HandleGenerateTSqlModelRequest(generateTSqlScriptParams, requestContext.Object);
+        await service.HandleGenerateTSqlModelRequest(generateTSqlScriptParams, requestContext.Object);
 
-            Directory.Delete(TSqlModelTestFolder);
-
-            Assert.True(service.projectModels.Value.Keys.Contains(generateTSqlScriptParams.ProjectUri), "Model was not stored under project uri");
-        }
+        Assert.True(service.projectModels.Value.Keys.Contains(generateTSqlScriptParams.ProjectUri), "Model was not stored under project uri");
```

Comment:
`Assert.That(service.projectModels.Value.Keys, Contains.Value(generateTSqlScriptParams.ProjectUri), "....")`